shared/messaging/proto/events.proto

```diff
@@ -5,3 +5,13 @@ package pocket;
 option go_package = "github.com/pokt-network/pocket/shared/messaging";
 
 message NodeStartedEvent {}
+
+message ConsensusNewHeightEvent {
+  uint64 height = 1;
+}
+
+message StateMachineTransitionEvent {
+  string event = 1;
+  string src = 2;
```

Comment:
What are `src` and `dst`?

Is it `prev_state` and `new_state`? If so, I'm okay with the names but just adding a comment `string src = 2; // previous state` would help